pkgs/by-name/tu/turborepo-remote-cache/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenvNoCC,
+  nodejs-slim_22,
+  pnpm_10,
+  jq,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "turborepo-remote-cache";
+  version = "2.6.0";
+
+  src = fetchFromGitHub {
+    owner = "ducktors";
+    repo = "turborepo-remote-cache";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-IBydX1+rPgFfgdov7pjUI7gyHBRXColcDaBOmzNNog8=";
+  };
+
+  pnpmDeps = pnpm_10.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-NlHQ3eJYFvZTGLzfDJPsBVlpMXOTeWdaTUukQQdLA+0=";
```

Please rebase to master and set this, otherwise the package would fail to build upon merging.

```suggestion
    fetcherVersion = 1;
    hash = "sha256-NlHQ3eJYFvZTGLzfDJPsBVlpMXOTeWdaTUukQQdLA+0=";
```